quickstarts/ext-hcp_vault/config.yml

```diff
@@ -0,0 +1,84 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: ext-hcp_vault
```

Comment:
```suggestion
slug: ext-hcp-vault
```

Reply:
I've performed the suggested changes, thanks